### PR TITLE
Add coverdb import

### PR DESCRIPTION
**Pull request by Drakulix**
Untested again ^^

### CODE REVIEW

File: src/coverdb.rs [hunk 1]
```diff
@@ -29,7 +31,7 @@ table! {
         Format -> Integer,
         Width -> Integer,
         Height -> Integer,
-        Data -> Nullable<Binary>,
+        Data -> Integer,
```
Comment (Drakulix) [inline src/coverdb.rs:33]: Urgh! Dont do that!
Just insert None.

File: src/coverdb.rs [hunk 2]
```diff
@@ -81,15 +83,13 @@ impl CoverDB {
...
-            let thumbnail: Vec<u8> = image.resize_to_fill(TEXTURE_WIDTH as u32, TEXTURE_HEIGHT as u32, FilterType::Triangle).into_rgb()
-                .pixels().flat_map(|pixel| pixel.channels()).copied().collect();
             diesel::insert_into(CoverThumbnail::table)
                 .values((
                     CoverThumbnail::ID.eq(id),
                     CoverThumbnail::Format.eq(TEXTURE_FORMAT),
-                    CoverThumbnail::Width.eq(TEXTURE_WIDTH),
-                    CoverThumbnail::Height.eq(TEXTURE_HEIGHT),
-                    CoverThumbnail::Data.eq(Some(&thumbnail)),
+                    CoverThumbnail::Width.eq(image.width() as i32),
```
Comment (Drakulix) [inline src/coverdb.rs:89]: Sure? Usdx code seems to store always the same value here.

File: src/coverdb.rs [hunk 3]
```diff
@@ -72,7 +72,9 @@ impl CoverDB {
                 .values((
                     Cover::Filename.eq(
                         cover.strip_prefix(&self.relative_to).with_context(|| format!("Cover '{}' is not relative to src_dir", cover.display()))?
-                        .to_str().with_context(|| format!("Unable to store filename '{}' in database", cover.display()))?),
+                        .to_str().with_context(|| format!("Unable to store filename '{}' in database", cover.display()))?
+                        .as_bytes()
+                    ),
```
Comment (Drakulix) [inline src/coverdb.rs]: How does the OsString look different in the database?
I mean this is not the correct type of the table field. Diesel generated the table macros from the an original table.db.

File: src/coverdb.rs (hunk 1 above)
Comment (finnboeger) [reply src/coverdb.rs:33]: Unfortunately that results in a different entry in the sqlite db. (NULL vs 0)

File: src/coverdb.rs (hunk 2 above)
Comment (finnboeger) [reply src/coverdb.rs:89]: When tested the stored size was always the same for the thumbnail and the cover

File: src/coverdb.rs (hunk 3 above)
Comment (finnboeger) [reply src/coverdb.rs]: By itself it resulted in the same output, but this allowed me to manipulate the bytes directly to achieve a result identical to usdx.

**Pull request merged**